src/scoring.js

```diff
@@ -0,0 +1,19 @@
+function scoring() {
+  this.score = 0
+}
+
+scoring.prototype.outcome = function(outcome){
+  return outcome
+}
+
+scoring.prototype._isstrike = function(x, y){
```

Comment:
Could _isstrike take a frame, rather than the individual rolls (to preserve encapsulation).